### ISSUE TITLE
Test case

### ISSUE THREAD

**Issue by YFLEEL** [NVIDIA/TensorRT#2395]
Hi，does Tensor8.2.0 provide test cases? Or do I need to implement all the samples provided in /root/samples to ensure the functionality of TensorRT?

**Comment by zerollzeng** [comment]
We have L0/L1 and QA tests for every release but they are not open-source.

**Comment by YFLEEL** [comment]
Thank you for your reply. The situation I'm facing is this: I need to build and test TensorRT on a new system to make sure it works fine on the new system. So wondering is there any way to get the L0/L1 and QA tests you mentioned (version 8.2.0)? If I can't get the test, do I need to run all the samples in the /root/samples to pass the test to ensure the realization of the system function? Looking forward to your reply!

**Comment by zerollzeng** [comment]
> So wondering is there any way to get the L0/L1 and QA tests you mentioned (version 8.2.0)?

Sorry they are not open-sourced.



> do I need to run all the samples in the /root/samples to pass the test to ensure the realization of the system function?

well I can guarantee that, but I would suggest quickly test with trtexec with a few models, and compare the performance with our official docker images.

May I ask what system you are going to build TensorRT?

**Comment by YFLEEL** [comment]
> well I can guarantee that, but I would suggest quickly test with trtexec with a few models, and compare the performance with our official docker images.

According to what you mean, can I implement the functional test of tensorrt as long as I run and pass the trtexec example in the samples and compare it with your official results? I see that trtexec is an example divided into C++ language, so does the python language need additional testing?

> May I ask what system you are going to build TensorRT?

The kernel of our system is 4.19.90-2209.1.0.0166.oe1.aarch64.

**Comment by zerollzeng** [comment]
Yes you can design and implement your functional test based on your need.

**Comment by YFLEEL** [comment]
Hi, according to your prompt, I am testing trtexec, but I have encountered the following problems, and I hope to get your answers:
1) The ```offical docker images``` you mentioned, can I find it directly or do I need to build and run trtexec on X86 to get it? If you can find it directly, is it convenient to provide the path to store it?
2) There are several cases involved in the operation of trtexec, and several files such as ```AlexNet_N2.prototxt``` ```GoogleNet_N2.prototxt``` that need to be used cannot be obtained by downloading ```https://developer.nvidia.com/nvidia-tensorrt-download```, but ```mnist .prototxt``` files are available. I would like to know how can I get these missing files?
Looking forward to hearing from you! Thank you very much~~

**Comment by zerollzeng** [comment]
> 1. The `offical docker images` you mentioned, can I find it directly or do I need to build and run trtexec on X86 to get it? If you can find it directly, is it convenient to provide the path to store it?

trtexec is under /usr/src/tensorrt/bin, there are also other samples.

> 2\. There are several cases involved in the operation of trtexec, and several files such as `AlexNet_N2.prototxt` `GoogleNet_N2.prototxt` that need to be used cannot be obtained by downloading `https://developer.nvidia.com/nvidia-tensorrt-download`, but `mnist .prototxt` files are available. I would like to know how can I get these missing files?
>     Looking forward to hearing from you! Thank you very much~~

We should document it on the readme, can you check it?

**Comment by YFLEEL** [comment]
```We should document it on the readme, can you check it?```
Hello, according to what you said, I went to read the readme carefully, but I still couldn't find the required ```AlexNet_N2.prototxt``` and ```GoogleNet_N2.prototxt```. Wondering if you could give a clearer hint?
Also, I found ```https://github.com/BVLC/caffe/tree/master/models/bvlc_alexnet``` based on relevant information I saw elsewhere, but I get an error when I run it, I wonder if I found the correct file?

**Comment by zerollzeng** [comment]
caffe parser is about to be deprecated, I would suggest to use onnx model, you can download it from https://github.com/onnx/models

**Comment by YFLEEL** [comment]
Hi, I have been testing the ```trtexec``` recently according to your prompts. There are some questions about DLA involved in the test.

I tested the models in Caffe format and ONNX format respectively, and found that the error ```Cannot create DLA engine would be reported```. I want to know whether the version of TensorRT 8.2.0 cannot call DLACore? The video card version we use is INVIDIA A100.

**Comment by zerollzeng** [comment]
A100 doesn't have DLA, it's only for embedded platform, like jetson or Drive.

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks, please reopen if you still have question, thanks!
